### PR TITLE
Added a middle bhop cap (slows players from skimming quicker)

### PR DESCRIPTION
**Pull request by mikeparker**
When you are over it, reduce speed by more. Currently set to 155% speed (620 for scout) and it has a lower pcfactor (0.4 rather than 0.65). This means downtrimping is good for one jump and reduces successive conc skim jumps without reducing the hard cap too much.

### CODE REVIEW

**Comment by AdamWillden** [comment]
I wonder if 0.3 pcfactor is a little harsh or 155% a little low. I approve of the idea overall though and it's a starting point. I need to get my machine set up to compile & test.

I'v been downloading SDK update and FF for days, I get 1.4Mb at best :(

Just going to link this with #36 as they're related

**Comment by mikeparker** [comment]
Without this change, a conc skim as scout, with a bit of turning / air accel between jumps goes:
conc (1400) -> 720 -> 666 -> 640 -> 620 -> 605 -> 596 -> bhop speed (6 jumps above bhop speed)

Similarly, a bhop into a single down trimp follows the same pattern, you get 6 jumps above bhop speed for a single downtrimp:
580 -> 720 (downtrimp) -> 666 -> 640 -> 620 -> 605 -> 596 -> bhop speed (6 jumps above bhop speed)

With this change, instead it goes:
conc (1400) -> 720 -> 608 -> bhop speed (2 jumps above bhop speed)

With pcfactor 0.5 instead of 0.3 it goes:
conc (1400) -> 720 -> 640 -> 600 -> bhop speed (3 jumps above bhop speed)

With pcfactor 0.4:
conc (1400) -> 720 -> 630 -> 600 -> bhop speed (3 jumps above bhop speed)

I guess i'd be happy anywhere between 0.3 and 0.5, I think 0.4 might actually feel the best. 
bear in mind that our old high cap (200%) was subject to the random 0.9 multiplier so it was actually 180%. With my other PR in, that leaves the caps at:
140% soft cap 35% reduction to anything over the limit
155% mid cap 60% reduction to anything over the limit
171% hard cap 100% reduction to anything over the limit

**Comment by mikeparker** [comment]
Ping @squeek502 for thoughts

**Comment by squeek502** [comment]
Wanted to test this locally first. Will comment later tonight.

**Comment by squeek502** [comment]
I like it. Doesn't feel like a brick wall and is a nice fairly subtle nerf to skimming.

:+1:

File: game_shared/ff/ff_gamemovement.cpp [hunk 1]
```diff
@@ -36,7 +36,7 @@
 //static ConVar sv_trimpmultiplier("sv_trimpmultiplier", "1.4", FCVAR_REPLICATED | FCVAR_CHEAT);
 #define SV_TRIMPMULTIPLIER 1.4f
 //static ConVar sv_trimpdownmultiplier("sv_trimpdownmultiplier", "1.2", FCVAR_REPLICATED | FCVAR_CHEAT);
-#define SV_TRIMPDOWNMULTIPLIER 1.2f
+#define SV_TRIMPDOWNMULTIPLIER 1.1f
```
Comment (mikeparker) [inline game_shared/ff/ff_gamemovement.cpp]: I don't think I meant to commit this line - i was experimenting with reducing the downtrimp amount

Comment (squeek502) [inline game_shared/ff/ff_gamemovement.cpp]: Yeah, I'd like to see this reverted before the PR is merged.

**Pull request merged**